docs/questions-and-answers.md

```diff
@@ -46,12 +46,11 @@ Anyone with the funds and technical know-how can create a subnet, or participate
 
 ### How does competition work in a subnet?
 
-The work to be performed by miners is set by the subnet ownerk in the form of the subnet's incentive mechanism. The miners compete to best perform the task, submitting their work to the validators. 
+The work to be performed by miners is set by the subnet creator in the form of the subnet's incentive mechanism. The miners compete to best perform the task, submitting their work to the validators. 
 
-The validators then rank the quality of the work done by the miners within the subnet.
-Miners harvest the TAO emitted to their subnet based on how well they perform, according to the aggregated scores of the validators. 
+The validators then rank the quality of the work done by the miners within the subnet. The aggregated scores of the validators determine the quantity of TAO emitted to each miner.
 
-At the same time, validators are also incentivized to do their best work, because their harvest depends on how well their miner scorings agree with the general consensus of other validators.
+At the same time, validators are also incentivized to do their best work, because their emissionsk depend on how well their miner scorings agree with the general consensus of other validators.
```

Comment:
```suggestion
At the same time, validators are also incentivized to do their best work, because their emissions depend on how well their miner scorings agree with the general consensus of other validators.
```